modules/API/API.hpp

```diff
@@ -192,6 +193,12 @@ class API : public Everest::ModuleBase {
     std::vector<std::thread> api_threads;
     bool running = true;
 
+    std::condition_variable evse_manager_cv;
+    std::mutex evse_manager_mux;
+    std::int8_t evse_manager_ready{0};
```

Comment:
According to the manifest, 128 connections can be configured to evse managers. In this case, the evse_manager_ready variable will overflow to -128 and it will be ready before they actually are.

Reply:
good spot - will use std::int16_t instead.

Reply:
@corneliusclaussen approach changed removing the evse_manager_ready counter.